database/nakadi/10_data/04_tables/01_event_type.sql

```diff
@@ -1,7 +1,10 @@
 CREATE TABLE IF NOT EXISTS zn_data.event_type (
   et_name varchar(255) NOT NULL PRIMARY KEY CHECK (et_name <> ''),
+  et_topic varchar(255) NOT NULL,
```

Comment:
We could also go for a more coupled solution and actually add it to the event type object also exposing it to users.